pages/legal/terms.vue

```diff
@@ -201,4 +201,8 @@ export default {
   @extend %card;
   padding: var(--spacing-card-sm) var(--spacing-card-lg);
 }
+
+a {
+  color: var(--color-link);
+}
```

Comment:
For consistencies sake, could this also be applied to the about page? It has links underlined but without a colour

Reply:
Added in ac33cc730d93aa5b5a75d69bc2288aa5ef40590d